Dockerfile

```diff
@@ -1,43 +1,45 @@
-    && apt-get -y update && apt-get -y upgrade \
-    && apt-get -y --no-install-recommends install \
-        cron \
-        logrotate \
-        monitoring-plugins \
-        supervisor \
-        nginx \
-        wallarm-node \
-        libnginx-mod-http-wallarm \
-        collectd-utils \
-        curl \
-        iptables \
-        bsdmainutils \
-        sudo \
-        socat \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* \
-    && chown -R wallarm:wallarm /var/lib/wallarm-tarantool \
-    && sed -i 's|session    required     pam_loginuid.so|session    optional     pam_loginuid.so|g' /etc/pam.d/cron
-
-RUN cp /usr/share/doc/libnginx-mod-http-wallarm/examples/wallarm-status.conf /etc/nginx/conf.d/
+ARG WLRM_FOLDER
+ARG TARGETPLATFORM
+
+RUN apk update && apk upgrade && apk add --no-cache bash runuser socat logrotate libgcc && rm -r /var/cache/apk/* && \
+  addgroup -S wallarm && \
+  adduser -S -D -G wallarm -h /opt/wallarm wallarm
+
+COPY --chown=wallarm:wallarm build/$TARGETPLATFORM/opt /opt
+COPY build/$TARGETPLATFORM/opt/wallarm/modules/${WLRM_FOLDER} /usr/lib/nginx/modules
+
+COPY build-scripts/check_sig.sh /opt/wallarm/check_sig.sh
+RUN /bin/sh -c '/opt/wallarm/check_sig.sh' && rm /opt/wallarm/check_sig.sh
+
+RUN sed -i \
```

Comment:
Can we implement this logic outside of Dockerfile, please?

Reply:
Moved to a separate script